### ISSUE TITLE
evaluation process is stucked

### ISSUE THREAD

**Issue by SeungHwi0613** [mit-han-lab/bevfusion#440]
when i learn this 
`torchpack dist-run -np 2 python tools/test.py configs/nuscenes/det/transfusion/secfpn/camera+lidar/swint_v0p075/convfuser.yaml pretrained/bevfusion-det.pth --eval bbox
`

`  top_proposals_class = top_proposals // heatmap.shape[-1]
[  ] 2/81, 0.5 task/s, elapsed: 4s, ETA:   145s/home/bev/bevfusion/mmdet3d/core/bbox/coders/transfusion_bbox_coder.py:96: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.post_center_range = torch.tensor(
[>>] 82/81, 7.9 task/s, elapsed: 10s, ETA:     0s`

process is stucked. i use 24+10gb gpu.

plus, when i tried 
`torchpack dist-run -np 2 python tools/test.py configs/nuscenes/det/centerhead/lssfpn/camera/256x704/swint/default.yaml pretrained/camera-only-det.pth --eval bbox`

it was working 

I want to know the reason of this....

**Comment by zhijian-liu** [comment]
Could you try running the evaluation on a single GPU to see if the same issue still persists? Thank you.
